libs/vre/shared/app-config/src/lib/app-config/app-config.service.spec.ts

```diff
@@ -26,55 +30,31 @@ describe('AppConfigService (dev)', () => {
         instrumentation: {
             environment: 'dev',
             dataDog: {
-                enabled: true,
-                applicationId: 'app_id',
-                clientToken: 'client_token',
-                site: 'site',
-                service: 'dsp-app',
+                enabled: false,
             },
             rollbar: {
-                enabled: true,
-                accessToken: 'rollbar_token',
+                enabled: false,
             },
         },
     };
 
-    const instrumentationConfig: DspInstrumentationConfig = {
-        environment: 'dev',
-        dataDog: {
-            enabled: false,
-            applicationId: 'app_id',
-            clientToken: 'client_token',
-            site: 'site',
-            service: 'dsp-app',
-        },
-        rollbar: {
-            enabled: false,
-            accessToken: 'rollbar_token',
-        },
-    };
-
     beforeEach(() => {
         TestBed.configureTestingModule({
             providers: [
                 {
                     provide: AppConfigToken,
                     useValue: devConfig,
                 },
-                {
-                    provide: DspInstrumentationToken,
-                    useValue: instrumentationConfig,
-                },
             ],
         });
         service = TestBed.inject(AppConfigService);
     });
 
-    it('should be created', () => {
+    test('should be created', () => {
```

Comment:
I prefer "it" because of https://stackoverflow.com/a/56072272

Reply:
it